Error when building tutorial application (video processing) on Ubuntu 17.04

```
Scanning dependencies of target video_app
[ 16%] Building CXX object CMakeFiles/video_app.dir/main.cc.o
In file included from /home/tobias/embb/doc/tutorial/application/main.cc:33:0:
/home/tobias/embb/doc/tutorial/application/include/frame_format_converter.h:35:6: error: use of enum ‘AVPixelFormat’ without previous declaration
 enum AVPixelFormat;
      ^~~~~~~~~~~~~
/home/tobias/embb/doc/tutorial/application/include/frame_format_converter.h:73:3: error: ‘AVPixelFormat’ does not name a type
   AVPixelFormat originalFormat;
   ^~~~~~~~~~~~~
/home/tobias/embb/doc/tutorial/application/main.cc: In function ‘void filter_opencl(AVFrame*)’:
/home/tobias/embb/doc/tutorial/application/main.cc:77:17: warning: ‘int avpicture_get_size(AVPixelFormat, int, int)’ is deprecated [-Wdeprecated-declarations]
   int n_bytes = avpicture_get_size(AV_PIX_FMT_RGB24, width, height);
                 ^~~~~~~~~~~~~~~~~~
In file included from /home/tobias/embb/doc/tutorial/application/include/ffmpeg.h:34:0,
                 from /home/tobias/embb/doc/tutorial/application/main.cc:35:
/usr/include/x86_64-linux-gnu/libavcodec/avcodec.h:5468:5: note: declared here
 int avpicture_get_size(enum AVPixelFormat pix_fmt, int width, int height);
     ^~~~~~~~~~~~~~~~~~
/home/tobias/embb/doc/tutorial/application/main.cc:77:67: warning: ‘int avpicture_get_size(AVPixelFormat, int, int)’ is deprecated [-Wdeprecated-declarations]
   int n_bytes = avpicture_get_size(AV_PIX_FMT_RGB24, width, height);
                                                                   ^
In file included from /home/tobias/embb/doc/tutorial/application/include/ffmpeg.h:34:0,
                 from /home/tobias/embb/doc/tutorial/application/main.cc:35:
/usr/include/x86_64-linux-gnu/libavcodec/avcodec.h:5468:5: note: declared here
 int avpicture_get_size(enum AVPixelFormat pix_fmt, int width, int height);
     ^~~~~~~~~~~~~~~~~~
CMakeFiles/video_app.dir/build.make:62: recipe for target 'CMakeFiles/video_app.dir/main.cc.o' failed
make[2]: *** [CMakeFiles/video_app.dir/main.cc.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/video_app.dir/all' failed
make[1]: *** [CMakeFiles/video_app.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/doc/tutorial/application/include/frame_format_converter.h
+++ b/doc/tutorial/application/include/frame_format_converter.h
@@ -29,10 +29,7 @@
 
 #include <embb/base/base.h>
 
-struct AVFrame;
-struct AVCodecContext;
-struct SwsContext;
-enum AVPixelFormat;
+#include "ffmpeg.h"
 
 enum ConversionType {
   TO_RGB,

--- a/doc/tutorial/application/src/frame_format_converter.cc
+++ b/doc/tutorial/application/src/frame_format_converter.cc
@@ -26,8 +26,6 @@
 
 #include "frame_format_converter.h"
 
-#include "ffmpeg.h"
-
 FrameFormatConverter::FrameFormatConverter()
   : toRGBCtx(nullptr)
   , toOriginalFormatCtx(nullptr) {

